CHANGELOG.md

```diff
@@ -4,6 +4,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [10.10.1]
+
+### Added
+- Added a new `ARIA_S1_COSEIS` job spec to JPL deployments.
```

Comment:
```suggestion
- Added a new `ARIA_S1_COSEIS` job spec to ARIA JPL deployments.
```

Reply:
Also, can you add it to the `hyp3-nisar-jpl` deployment in `deploy-enterprise.yml`? It doesn't have a corresponding test deployment, but it is one of the production ARIA deployments.